public/app/features/profile/UserPreferences.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { PureComponent } from 'react';
+import { hot } from 'react-hot-loader';
+import { connect } from 'react-redux';
+import { StoreState, OrgUser } from 'app/types';
+import { loadUser, clearUser } from 'app/core/components/UserEdit/state/actions';
+import { getNavModel } from 'app/core/selectors/navModel';
+import { NavModel } from '@grafana/data';
+import Page from 'app/core/components/Page/Page';
+import UserProfile from 'app/core/components/UserEdit/UserProfile';
+import UserSessions from 'app/core/components/UserEdit/UserSessions';
+import UserTeams from 'app/core/components/UserEdit/UserTeams';
+import SharedPreferences from 'app/core/components/SharedPreferences/SharedPreferences';
+
+export interface Props {
+  navModel: NavModel;
+  user: OrgUser;
+  loadUser: typeof loadUser;
+  clearUser: typeof clearUser;
+}
+
+export class UserPreferences extends PureComponent<Props> {
```

Comment:
UserProfilePage ? it's the default landing page for the user profile, feels like a better name